src/PKSim.Core/Repositories/ITransporterContainerTemplateRepository.cs

```diff
@@ -36,5 +36,10 @@ public interface ITransporterContainerTemplateRepository : IStartableRepository<
       /// <param name="speciesName">Species where the transporter will be defined</param>
       /// <param name="containerName">Concrete transporter location (e.g Kidney, Liver etc...)</param>
       IEnumerable<TransporterContainerTemplate> TransportersFor(string speciesName, string containerName);
+
+      /// <summary>
+      /// Returns the name of all transporters defined in the PKSim Database (template transporters and their synonyms)
+      /// </summary>
+      IReadOnlyList<string> AllTransporterNames { get; }
```

Comment:
nothing to do with the issue but we added a bunch of molecules from the db and those were msissing